recipes/wagyu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagyu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0e4b78913e64083ca865f07e6aee30a24988890fc383f3003456287ed45d7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pybind11 >=2.6.1
```

Comment:
is space between name of the package and version required? I'm asking because not so familiar with syntax

Reply:
Yep, this is fine for conda recipes